### PR TITLE
Feat/user login

### PR DESCRIPTION
**Pull request by learnforward2023**
## Why

Fixes: #1 

## What

- [Feat: add login feature](https://github.com/learnforward2023/project-server/pull/27/commits/2f14f64d79a1f3eb2d9b8246c834edd12c9c035a)
- [Return token when sign up successfully](https://github.com/learnforward2023/project-server/pull/27/commits/65deae98f5ca1b72cd3f9f25458e5cd0500c2acc)
- [Add message when sign in successfully](https://github.com/learnforward2023/project-server/pull/27/commits/93dee3a37fd2267999029eb9a3d4580b3c771331)

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ module.exports = {
     ],
     quotes: ['error', 'single'],
     'template-curly-spacing': 'error',
+    "@typescript-eslint/strict-boolean-expressions": "off",
```
Comment (learnforward2022) [inline .eslintrc.js:81]: 🚫 **[eslint]** <[quotes](https://eslint.org/docs/latest/rules/quotes)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Strings must use singlequote.
```suggestion
    '@typescript-eslint/strict-boolean-expressions': "off",
```

Comment (learnforward2022) [inline .eslintrc.js:81]: 🚫 **[eslint]** <[@typescript-eslint/quotes](https://typescript-eslint.io/rules/quotes)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Strings must use singlequote.
```suggestion
    '@typescript-eslint/strict-boolean-expressions': "off",
```

Comment (learnforward2022) [inline .eslintrc.js:81]: 🚫 **[eslint]** <[quotes](https://eslint.org/docs/latest/rules/quotes)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Strings must use singlequote.
```suggestion
    "@typescript-eslint/strict-boolean-expressions": 'off',
```

Comment (learnforward2022) [inline .eslintrc.js:81]: 🚫 **[eslint]** <[@typescript-eslint/quotes](https://typescript-eslint.io/rules/quotes)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Strings must use singlequote.
```suggestion
    "@typescript-eslint/strict-boolean-expressions": 'off',
```

**Review by NQTrung08 — APPROVED** [review]
LGTM

File: .eslintrc.js [hunk 2]
```diff
@@ -78,6 +78,7 @@ module.exports = {
     ],
     quotes: ['error', 'single'],
     'template-curly-spacing': 'error',
+    '@typescript-eslint/strict-boolean-expressions': "off",
```
Comment (learnforward2022) [inline .eslintrc.js:81]: 🚫 **[eslint]** <[quotes](https://eslint.org/docs/latest/rules/quotes)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Strings must use singlequote.
```suggestion
    '@typescript-eslint/strict-boolean-expressions': 'off',
```

Comment (learnforward2022) [inline .eslintrc.js:81]: 🚫 **[eslint]** <[@typescript-eslint/quotes](https://typescript-eslint.io/rules/quotes)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Strings must use singlequote.
```suggestion
    '@typescript-eslint/strict-boolean-expressions': 'off',
```

**Pull request merged**